src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -341,6 +341,22 @@ public string GetHelpMaml(string helpItemName, string path)
         #endregion
 
         #region CmdletProvider members
+        // OneDrive placeholder support (issue #8315)
+#if !UNIX
+        [DllImport("ntdll.dll")]
+        static extern sbyte RtlQueryProcessPlaceholderCompatibilityMode();
+        [DllImport("ntdll.dll")]
+        static extern sbyte RtlSetProcessPlaceholderCompatibilityMode(sbyte pcm);
+
+        const sbyte PHCM_APPLICATION_DEFAULT = 0;
+        const sbyte PHCM_DISGUISE_PLACEHOLDER = 1;
+        const sbyte PHCM_EXPOSE_PLACEHOLDERS = 2;
+        const sbyte PHCM_MAX = 2;
+        const sbyte PHCM_ERROR_INVALID_PARAMETER = -1;
+        const sbyte PHCM_ERROR_NO_TEB = -2;
```

Comment:
Seems like you could define an enum type for this purpose? The PHCM_MAX constant would either be implicit or a duplicate/Alias definition to the EXPOSE_PLACEHOLDERS value.

Reply:
I do not see the need.

Please use `internal const`.

Reply:
An enum indeed seems like overkill given that I need only two of the values I got from `ntifs.h`

Added `internal`